### PR TITLE
Release 5.3.0

### PR DESCRIPTION
**Pull request by mgreisen**
## Summary of the Pull Request

_What is this about?_

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

**Review by chkeita — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.3.0
+### Added
+* Agent: Add a compiler flag to generate debug info for the windows-libfuzzer load library test target. [#1684](https://github.com/microsoft/onefuzz/pull/1684)
+* Agent: Add a rust crate to debug missing dynamic library errors on Windows. [#1713](https://github.com/microsoft/onefuzz/pull/1713)
+* Agent: Add support for detecting missing dynamic libraries on Linux.. [#1718](https://github.com/microsoft/onefuzz/pull/1718)
```
Comment (ranweiler) [inline CHANGELOG.md:11]:
```suggestion
* Agent: Add support for detecting missing dynamic libraries on Linux. [#1718](https://github.com/microsoft/onefuzz/pull/1718)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.3.0
+### Added
+* Agent: Add a compiler flag to generate debug info for the windows-libfuzzer load library test target. [#1684](https://github.com/microsoft/onefuzz/pull/1684)
```
Comment (ranweiler) [inline CHANGELOG.md:9]:
```suggestion
* Agent: Add a compiler flag to generate debug info for the `windows-libfuzzer` load library test target. [#1684](https://github.com/microsoft/onefuzz/pull/1684)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Add a rust crate to debug missing dynamic library errors on Windows. [#1713](https://github.com/microsoft/onefuzz/pull/1713)
```
Comment (ranweiler) [inline CHANGELOG.md:10]:
```suggestion
* Agent: Add a Rust crate to debug missing dynamic library errors on Windows. [#1713](https://github.com/microsoft/onefuzz/pull/1713)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Add support for detecting missing dynamic libraries on Linux.. [#1718](https://github.com/microsoft/onefuzz/pull/1718)
+* Service: Connect the auto scaling diagnostics to the log analytics workspace. [#1708](https://github.com/microsoft/onefuzz/pull/1708)
+* Service: Handle the situation where a VM scale set instance is destroyed  before we have removed scale-in protection. [#1719](https://github.com/microsoft/onefuzz/pull/1719)
```
Comment (ranweiler) [inline CHANGELOG.md:13]:
```suggestion
* Service: Handle the situation where a VM scale set instance is destroyed before we have removed scale-in protection. [#1719](https://github.com/microsoft/onefuzz/pull/1719)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+* Service: Port ARM templates to Bicep. [#1724](https://github.com/microsoft/onefuzz/pull/1724), [#1732](https://github.com/microsoft/onefuzz/pull/1732)
+* Service: Initial changes to port the service from python to C#. [#1734](https://github.com/microsoft/onefuzz/pull/1734), [#1733](https://github.com/microsoft/onefuzz/pull/1733), [#1736](https://github.com/microsoft/onefuzz/pull/1736), [#1737](https://github.com/microsoft/onefuzz/pull/1737), [#1738](https://github.com/microsoft/onefuzz/pull/1738), [#1742](https://github.com/microsoft/onefuzz/pull/1742), [#1744](https://github.com/microsoft/onefuzz/pull/1744), [#1749](https://github.com/microsoft/onefuzz/pull/1749), [#1750](https://github.com/microsoft/onefuzz/pull/1750), [#1753](https://github.com/microsoft/onefuzz/pull/1753), [#1755](https://github.com/microsoft/onefuzz/pull/1755), [#1760](https://github.com/microsoft/onefuzz/pull/1760), [#1761](https://github.com/microsoft/onefuzz/pull/1761), [#1762](https://github.com/microsoft/onefuzz/pull/1762), [#1765](https://github.com/microsoft/onefuzz/pull/1765)
```
Comment (ranweiler) [inline CHANGELOG.md:17]:
```suggestion
* Service: Initial changes to port the service from Python to C#. [#1734](https://github.com/microsoft/onefuzz/pull/1734), [#1733](https://github.com/microsoft/onefuzz/pull/1733), [#1736](https://github.com/microsoft/onefuzz/pull/1736), [#1737](https://github.com/microsoft/onefuzz/pull/1737), [#1738](https://github.com/microsoft/onefuzz/pull/1738), [#1742](https://github.com/microsoft/onefuzz/pull/1742), [#1744](https://github.com/microsoft/onefuzz/pull/1744), [#1749](https://github.com/microsoft/onefuzz/pull/1749), [#1750](https://github.com/microsoft/onefuzz/pull/1750), [#1753](https://github.com/microsoft/onefuzz/pull/1753), [#1755](https://github.com/microsoft/onefuzz/pull/1755), [#1760](https://github.com/microsoft/onefuzz/pull/1760), [#1761](https://github.com/microsoft/onefuzz/pull/1761), [#1762](https://github.com/microsoft/onefuzz/pull/1762), [#1765](https://github.com/microsoft/onefuzz/pull/1765)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+* Service: Make sure the scale set nodes are unable to accept work while in the `setup` state. [#1731](https://github.com/microsoft/onefuzz/pull/1731)
+
+### Changed
+* Agent: Reduce the logging level down from `warn` to `debug` when we are unable to parse an asan log. [#1705](https://github.com/microsoft/onefuzz/pull/1705)
```
Comment (ranweiler) [inline CHANGELOG.md:21]:
```suggestion
* Agent: Reduce the logging level down from `warn` to `debug` when we are unable to parse an ASan log. [#1705](https://github.com/microsoft/onefuzz/pull/1705)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.3.0
+### Added
+* Agent: Add a compiler flag to generate debug info for the windows-libfuzzer load library test target. [#1684](https://github.com/microsoft/onefuzz/pull/1684)
+* Agent: Add a rust crate to debug missing dynamic library errors on Windows. [#1713](https://github.com/microsoft/onefuzz/pull/1713)
+* Agent: Add support for detecting missing dynamic libraries on Linux.. [#1718](https://github.com/microsoft/onefuzz/pull/1718)
+* Service: Connect the auto scaling diagnostics to the log analytics workspace. [#1708](https://github.com/microsoft/onefuzz/pull/1708)
+* Service: Handle the situation where a VM scale set instance is destroyed  before we have removed scale-in protection. [#1719](https://github.com/microsoft/onefuzz/pull/1719)
+* Service: Add support for auto scaling in the CLI.[#1717](https://github.com/microsoft/onefuzz/pull/1717)
```
Comment (tevoinea) [inline CHANGELOG.md:14]:
```suggestion
* Service: Add support for auto scaling in the CLI. (New scale sets will automatically be created with auto scaling enabled)[#1717](https://github.com/microsoft/onefuzz/pull/1717)
```

File: CHANGELOG.md (hunk 5 above)
Comment (mgreisen) [reply CHANGELOG.md]: I've made this change.

File: CHANGELOG.md (hunk 7 above)
Comment (mgreisen) [reply CHANGELOG.md]: I've added this sentence.

**Pull request merged**